packages/govuk-frontend/src/govuk/components/globals.test.js

```diff
@@ -37,6 +37,7 @@ describe('GOV.UK Frontend', () => {
         'Checkboxes',
         'ErrorSummary',
         'ExitThisPage',
+        'GOVUKFrontendError',
```

Comment:
Related to https://github.com/alphagov/govuk-frontend/pull/4030/files#r1284215663

Should we not export our errors in the default export, we'll need to manage Terser mangling:

https://github.com/alphagov/govuk-frontend/blob/91d0a5d8b694875b34eb4be52ecf155f8b3701d0/packages/govuk-frontend/rollup.release.config.mjs#L31-L33

Reply:
Can you expand on what we'll need to manage on the Terser side if we don't export the errors internally? 

If it's about the error naming after mangling, [the hardcoded `name` property is there to handle the situation](https://github.com/alphagov/govuk-frontend/pull/4030#discussion_r1280921668) (especially as we can't guarantee that library users will take the same step as us towards not mangling the name of the errors/our exports).

Reply:
Yeah that `mangle.reserved` option

But if the mangled error class name doesn't appear in stack traces then all good

Reply:
Added a comment about mangling on this PR: https://github.com/alphagov/govuk-frontend/pull/4075.

The name is clear in the logged message, but mangled in the stacktrace. I think mangling is worth its own discussion, as it may be worth disabling it altogether rather than keep maintaining an evergrowing list of names to not mangle for our `min.js` file.

Reply:
And opened https://github.com/alphagov/govuk-frontend/issues/4076 to explore our options.